build/BUILD.simdutf

```diff
@@ -3,10 +3,14 @@ cc_library(
     srcs = ["simdutf.cpp"],
     hdrs = ["simdutf.h"],
     copts = select({
+        "@v8//bazel/config:is_clang": ["-std=c++20"],
```

Comment:
Nit: we already specify these in .bazelrc, and as I understand simdutf is able to set SIMDUTF_ATOMIC_REF on its own if the underlying feature is available (which is the case with our updated compiler); no changes to this file should be needed.